superset-frontend/src/cccs-viz/README.md

```diff
@@ -30,29 +30,17 @@ Modified instructions from
 https://superset.apache.org/docs/installation/building-custom-viz-plugins
 
 ```bash
-cd superset/superset-frontend/cccs-viz/plugins
+cd superset/superset-frontend/src/cccs-viz/plugins
```

Comment:
I think this is the opportunity to change this: cccs-viz to cccs-visualizations.

Reply:
since we are going to contribute both visualizations and native filters I was thinking of maybe just calling it cccs.

but I have not yet moved the native filter ipv4 into it yet